### DISCUSSION TITLE
Plotting Time Frames getting longer and longer

### DISCUSSION THREAD

**Discussion by number435398** [Chia-Network/chia-blockchain#6584] [Support]
I have been plotting on two separate computers and I'm finding that on both computers that, over time, they start taking longer and longer to plot. I run many in parallel and keep extending the times between them and it still gets slower and slower. On one computer, it was about 10 hrs for a plot, now its over 20 while using nvme drives.  Same on the second computer.  It seems to correlate with very little CPU usage.  Like the first 20 plots were fine on both computers, and after that it all changed. 

One computer is now running over 40 plots simultaneously because of how slow its gotten (since my time delays begin to overlap too much) and yet its near constantly using no more than about 4 CPU threads worth of processing and everything is taking 2-3 times as long as it did several days ago.  I'm using Windows 10 on both computers. When comparing speeds, I'm talking about using the exact same drives and hardware for the faster plots and when they got slower.

**Comment by XIII39** [comment]
Keep an eye on your SSD temps.. As the drives get too hot they slow down, significantly. 50 degrees C is twice as fast as 70 - on average.
Slap a thermal pad on top of the drive, add a heatsink or just an extra fan.

**Reply by number435398** [reply]
My 2nd computer has temps on the nvme drives of under 40c, some even under 30c (as they have heatsink/fan/thermal tape).  The room both my computers are run in is typically under 80F (26c).  I really think the chia program is inadvertently building up something that's slowing it down or Windows 10 is, but I can't imagine what it'd be for the latter.

**Comment by BrandtH22** [comment]
@number435398 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
